### PR TITLE
Expose resourceLimits for instances of pods and apps

### PR DESCRIPTION
**Pull request by timcharper**
JIRA Issues: MARATHON-8749

### CODE REVIEW

**Comment by deveshk0** [comment]
any idea when it will available?

**Comment by timcharper** [comment]
Planning on merging this for DC/OS 2.2

File: src/main/scala/mesosphere/marathon/core/pod/PodDefinition.scala [hunk 1]
```diff
@@ -148,4 +149,20 @@ object PodDefinition {
...
+    if ((cpusLimit == 0d) && (memLimit == 0d)) {
+      None
+    } else {
+      Some(
+        ResourceLimits(
+          cpus = (if (cpusLimit == 0d) None else Some(cpusLimit)),
+          mem = (if (memLimit == 0d) None else Some(memLimit))
+        )
+      )
+    }
```
Comment (pierrebeitz) [inline src/main/scala/mesosphere/marathon/core/pod/PodDefinition.scala:165]: only now i realize that this won't help the frontend much.  
i was looking to have marathon factor in the number of currently running instances as well as the executor needs.

here are the current calculations: https://github.com/dcos/dcos-ui/blob/69d94db858d6c406c43846db586b57ca09100de1/plugins/services/src/js/columns/resourceLimits.ts#L34

i also think that we should make a difference between `None` and `0` as we recently encountered quota issues due to exactly not making that difference: https://jira.d2iq.com/browse/COPS-6589.

**Review by pierrebeitz — CHANGES_REQUESTED** [review]
I think we should not pursue implementing this if we don't want to factor in exector-resources and number of instances, as else it does not help clients much with that calculations.

I apologize for not properly articulating this in the first place.

**Pull request closed without merging**